.pipelines/rp-int-deploy.yml

```diff
@@ -0,0 +1,88 @@
+
+    steps:
+# setup
+      - template: ./templates/template-checkout.yml
+      - template: ./templates/template-az-cli-login.yml # does it have az CLI alraedy?
+        parameters:
+          azureDevOpsJSONSPN: $(aro-v4-e2e-devops-spn)
+      - template: ./templates/rp-int-deployment/template-download-secrets.yml
+      - template: ./templates/rp-int-deployment/template-generate-rp-dev-config.yml
+        parameters:
+          location: ${{ parameters.location }}
+          resourceGroup: ${{ parameters.resourceGroup }}
+          databaseAccountName: ${{ parameters.databaseAccountName }}
+          keyvaultPrefix: ${{ parameters.keyvaultPrefix }}
+          aroImage: ${{ parameters.aroImage }}
+          fluentbitImage: ${{ parameters.fluentbitImage }}
+      - script: make deploy # use different target which will succeed and won't fail
+        name: semi-deploy-aro-rp-int
+        displayName: ⚙️ Run pre-deplyments tasks prior to AKS
+      - script: source ./hack/devtools/deploy-shared-env.sh && deploy_vpn_for_dedicated_rp
+        name: deploy-vpn-gateway
+        displayName: ⚙️ Deploy VPN Gateway
+      - script: source ./hack/devtools/deploy-shared-env.sh && deploy_aks_dev
+        name: deploy-aks-dev
+        displayName: ⚙️ Deploy AKS
+      - template: ./templates/rp-int-deployment/template-install-hive.yml
+        parameters:
+          location: ${{ parameters.location }}
+      - template: ./templates/rp-int-deployment/template-mirror-ocp-images.yml
+        parameters:
+          dst_acr_name: ${{ parameters.dst_acr_name }}
+          tag: ${{ parameters.ocpVersionTag }}
+          genevamdmImageTag: ${{ parameters.genevamdmImageTag }}
+          genevamdsdImageTag: ${{ parameters.genevamdsdImageTag }}
+      - template: ./templates/rp-int-deployment/template-push-aro-fluenbit-to-acr.yml
+        parameters:
+          fluentbitImageTag: ${{ parameters.fluentbitImageTag }
+          fluentbitImage: ${{ parameters.fluentbitImage }
+      - template: ./templates/rp-int-deployment/template-dns-domain-and-certificate-keyvault.yml
+      - template: ./templates/rp-int-deployment/template-delete-vmss.yml
+        parameters:
+          resourceGroup: ${{ parameters.resourceGroup }}
+          vmssName: rp-vmss-${{ parameters.gitCommit }}
+      - template: ./templates/rp-int-deployment/template-delete-vmss.yml
+        parameters:
+          resourceGroup: $(UNIQUE_IDENTIFIER)-gwy-${{ parameters.location }}
+          vmssName: gateway-vmss-${{ parameters.gitCommit }}
+      - script: make deploy
+        name: full-deploy-aro-rp-int
+        displayName: ⚙️ Fully deploy ARO RP integreation
+      - script: |
+          source ./hack/devtools/deploy-shared-env.sh
+          deploy_oic_for_dedicated_rp
+        name: create-sa-role
+        displayName: ⚙️ Create storage account and role assignment
+## few more steps would be added
+
+# cleanup
+      - template: ./templates/template-clean-subscription.yml
```

Comment:
Isn't this the template we use in the pipeline that cleans dev clusters in the RH tenant? If so, why is it being used here?

Reply:
I though it could be used for cleaning all the resources the pipeline is currently creating. But I guess it should be done in a different way..